src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,17 @@
+package Duke;
```

Comment:
Package name should be in lowercase. Could maybe refer to the coding standards guideline.